runner.py

```diff
@@ -8,7 +8,18 @@
 
     is_dummy = False
     args = ' '.join(sys.argv[1:])
+    if 'subdir' in config:
+        if not 'suite' in config:
+            print 'Test subdirectory present in config without specific test suite to run.'
+            sys.exit(1)
+        subdir = config['subdir']
+        suite = config['suite']
+    else:
+        subdir = None
+        suite = None
```

Comment:
These `subdir` and `suite` declarations could be removed. After making this check, we should have `subdir=config.get('subdir'), suite=config.get('suite')` in the constructor below.